apps/web/src/components/ParticipantWindow/ParticipantWindow.tsx

```diff
@@ -54,6 +55,7 @@ export default function ParticipantWindow() {
   return (
     <aside className={clsx(classes.participantWindowContainer, { [classes.hide]: !appState.isParticipantWindowOpen })}>
```

Comment:
I think we'll need to add some CSS to make this list scrollable when it gets long, but we can probably handle that in the Viewers List PR, since the Speakers List and Viewers List will both need to be able to scroll within the ParticipantWindow.